src/views/LoginForm/loginFormConstants.json

```diff
@@ -1,12 +1,12 @@
 [
     {
-        "placeholder": "Enter your email",
+        "placeholder": "@ Enter your email",
```

Comment:
I think you can change all the placeholders to be simply the noun form. E.g.: "Email", "Password", rather than a sentence. Seems to be the norm as well for most websites.
Idem for other placeholder

Reply:
I second this ^.

Just keep the noun